docs/development.md

```diff
@@ -281,12 +282,55 @@ I0511 10:15:29.593106 244858 hello_world.cc:57] Response: HELLO AGAIN WORLDS!
 
 ### Signing Oak Wasm modules
 
-Oak Wasm modules can be signed using [Ed25519](https://ed25519.cr.yp.to/)
-scheme.
+As described in [Oak Concepts](/docs/concepts.md#labels) Oak Wasm modules can be
+signed using [Ed25519](https://ed25519.cr.yp.to/) scheme.
 
-In order create a signature `hello_world.sign` for a Wasm module
-`hello_world.wasm` use the following script:
+If you are reviewing an Oak module (for example `hello_world.wasm`) and want to
+create a signature for it, first you need a private/public key pair. It can be
+generated using `oak_sign` utility:
 
 ```bash
-./scripts/sign_module -i examples/hello_world/bin/hello_world.wasm -o examples/hello_world/bin/hello_world.sign
+./scripts/oak_sign generate \
+  --private-key=examples/certs/ed25519/test.key \
+  --public-key=examples/certs/ed25519/test.pub
 ```
+
+It generates two key files (`test.key` and `test.pub`) that are encoded using
+[PEM](https://tools.ietf.org/html/rfc1421) format.
+
+Then a Wasm module can be signed using the following command:
+
+```bash
+./scripts/oak_sign sign \
+  --private-key=examples/certs/ed25519/test.key \
+  --input-file=examples/hello_world/bin/hello_world.wasm \
+  --signature=examples/hello_world/bin/hello_world.sign
+```
+
+This command generates a signature file `hello_world.sign` contains both a
+public key and a Wasm modules signature (also encoded with PEM).
+
+Wasm module signatures should be passed to `oak_loader` by an Oak server
+maintainer using the `--signatures-manifest=signature.toml`, where
```

Comment:
Previously it sounded like the term is Oak application maintainer, but here it says Oak server maintainer, though I guess you meant the same thing in both, right?

Reply:
Changed to `Oak operator` (similar to the life cycle)